lib/src/common/pulls_service.dart

```diff
@@ -184,3 +196,9 @@ class PullRequestsService extends Service {
     );
   }
 }
+
+enum MergeMethod {
```

Comment:
Can you add a dartdoc with a link to the GitHub reference

Reply:
Yeah I can do that.